apk/Dockerfile-alpine

```
@@ -1,5 +1,5 @@
 # Need to use version number so that it gets updated here and triggers a build
-FROM alpine:3.16.2
+FROM alpine:3.16.3
```

3.17 came out, should we use the latest?

```suggestion
FROM alpine:3.17
```

---

No, we want to maintain parity with Geodesic, and Alpine minor releases often break something, so we want to wait until we have time to deal with the breakage and/or need the update.